docs/version0.13.md

```diff
@@ -41,15 +43,29 @@ OpenJ9 release 0.13.0 supports OpenJDK 12, which is available from the AdoptOpen
 
 OpenJDK 12 with Eclipse OpenJ9 is not a long term support (LTS) release.
 
-Although it is possible to build an OpenJDK 8, or other versions, with OpenJ9 0.13.0, testing at the project is not complete and therefore support is not available.
+The latest builds of OpenJDK with OpenJ9 for Java 8 and 11 at the AdoptOpenJDK community are for Eclipse OpenJ9 release 0.12.0. Features mentioned in these release notes are not available in these builds. Although it might be possible to build an OpenJDK 8 or OpenJDK 11 with OpenJ9 0.13.0, testing at the project is not complete and therefore support for any of these features is not available.
 
-To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md)
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
+
+### Support for OpenSSL 1.0.2
+
+![Start of content that applies only to Java 12](cr/java12.png)
+
+OpenSSL cryptographic support is extended to include OpenSSL 1.0.2 for the Digest, CBC, GCM, and RSA algorithms. Support is enabled by default. On Linux and AIX platforms, the OpenSSL libraries are expected to be available on the system path. For more information about cryptographic acceleration with OpenSSL, see [Cryptographic operations](introduction.md#cryptographic-operations).
+
+In addition, support for the OpenSSL 1.1.X Digest algorithm is re-enabled in this release following the resolution of issue [#4530](https://github.com/eclipse/openj9/issues/4530).
```

Comment:
This should say "support for the OpenSSL Digest algorithm" and not "support for the OpenSSL **1.1.X** Digest algorithm"

Reply:
Fixed